resources/definitions/fdmprinter.def.json

```diff
@@ -3379,6 +3379,30 @@
                     "settable_per_mesh": false,
                     "settable_per_extruder": true
                 },
+                "support_skip_some_zags":
+                {
+                    "label": "Skip Some ZigZags Connections",
+                    "description": "Skip some ZigZags connections to make the support structure easier to break.",
+                    "type": "bool",
+                    "default_value": false,
+                    "enabled": "support_enable and (support_pattern == 'zigzag')",
+                    "limit_to_extruder": "support_infill_extruder_nr",
+                    "settable_per_mesh": false,
+                    "settable_per_extruder": true
+                },
+                "support_zag_skip_count":
+                {
+                    "label": "ZigZag Connection Skip Count",
```

Comment:
Discussed with Ruben and we think if we create a parent setting "support_zag_skip_count_per_mm" which is the "frequency" mentioned above, then this one can be called "Support Line Connection Skip Chunk Size" or similar since this will be automatically calculated from the "frequency".